Description: Set BOOST_FILESYSTEM_VERSION=2
 This prevents FTBFS with boost >= 1.46, which sets
 BOOST_FILESYSTEM_VERSION=3 by default.
Author: Evgeni Golov <evgeni@debian.org>
Bug-Debian: http://bugs.debian.org/625109

Index: performous-0.7.0/CMakeLists.txt
===================================================================
--- performous-0.7.0.orig/CMakeLists.txt	2012-11-12 23:14:08.000000000 +0100
+++ performous-0.7.0/CMakeLists.txt	2012-11-19 19:48:21.840826743 +0100
@@ -15,6 +15,8 @@
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
+add_definitions(-DBOOST_FILESYSTEM_VERSION=2)
+
 # Add a sensible build type default and warning because empty means no optimization and no debug info.
 if(NOT CMAKE_BUILD_TYPE)
 	message("WARNING: CMAKE_BUILD_TYPE is not defined!\n         Defaulting to CMAKE_BUILD_TYPE=RelWithDebInfo. Use ccmake to set a proper value.")
